azure-pipelines.yml

```
@@ -116,16 +144,16 @@ jobs:
       frameworks: # for private jobs we want to benchmark .NET Core 3.0 only
         - netcoreapp3.0
         
-# Ubuntu 1604 x64 micro benchmarks, public correctness job
+# Ubuntu 1804 x64 micro benchmarks, public correctness job
 - ${{ if eq(variables['System.TeamProject'], 'public') }}:
   - template: /eng/performance/benchmark_jobs.yml
     parameters:
       osName: ubuntu
-      osVersion: 1604
+      osVersion: 1804
       kind: micro
       architecture: x64
       pool: Hosted Ubuntu 1604
```

nit: should the name of the pool be updated as well?

```suggestion
      pool: Hosted Ubuntu 1804
```

---

There is no Hosted Ubuntu 1804 queue, I believe. I had changed that, but it broke it. It also does not matter which hosted queue we are on, as that's just where we trigger the helix runs from (we don't build on build machines).